vue/components/forms/personalization-form/initials-images.vue

```diff
@@ -215,6 +215,12 @@ export const InitialsImages = {
             if (this.imageBorderRadius) base["border-radius"] = `${this.imageBorderRadius}`;
             if (this.imageObjectFit) base["object-fit"] = this.imageObjectFit;
             return base;
+        },
+        openExternallyFeature() {
+            const features = this.$store.state.features || {};

```

Comment:
We should avoid this strategy as the "features" concept is a White idea and not part of the ripe-commons domain. I believe we'll have to do this completely inside white codebase.

Reply:
I had that idea initially and changed the clients of this component to pass the correct prop value. But that implies changing white where it uses this component as well as all retail vendors and plugins.

It does make more sense because the component shouldn't know about the concept of features.